src/electron/routing_service.ts

```diff
@@ -100,27 +126,31 @@ export class WindowsRoutingService implements RoutingService {
       });
 
       this.ipcConnection.on('error', (e: NetError) => {
-        if (retry) {
-          console.info(`bouncing OutlineService (${e.errno})`);
-          sudo.exec(SERVICE_START_COMMAND, {name: 'Outline'}, (sudoError, stdout, stderr) => {
-            if (sudoError) {
-              // Yes, this seems to be the only way to tell.
-              if ((typeof sudoError === 'string') &&
-                  sudoError.toLowerCase().indexOf('did not grant permission') >= 0) {
-                return reject(new errors.NoAdminPermissions());
-              } else {
-                // It's unclear what type sudoError is because it has no message
-                // field. toString() seems to work in most cases, so use that -
-                // anything else will eventually show up in Sentry.
-                return reject(new errors.ConfigureSystemProxyFailure(sudoError.toString()));
+        if (isWindows) {
```

Comment:
I think you can avoid increasing the indentation level with `if (isWindows && retry)`